src/account/operations/participation/mod.rs

```diff
@@ -176,3 +171,20 @@ impl AccountHandle {
         Ok(())
     }
 }
+
+fn is_valid_participation_output(output: &Output) -> bool {
+    // Only basic outputs can be participation outputs.
+    if let Output::Basic(basic_output) = &output {
+        // Valid participation outputs can only have the AddressUnlockCondition
+        let [UnlockCondition::Address(_)] = basic_output.unlock_conditions().as_ref() else{
+            return false;
+        };
+        if let Some(tag) = basic_output.features().tag() {
+            tag.tag() == PARTICIPATION_TAG.as_bytes()
```

Comment:
In one of the filters, were were checking that we have some metadata, do we not care anymore ?

Reply:
Good catch, we should still check that before requesting, because without metadata it shouldn't be participating anywhere